functional_tests/scylla_operator/test_functional.py

```diff
@@ -178,7 +179,6 @@ def test_drain_terminate_decommission_add_node_kubernetes(db_cluster):
 #       - '2.3.x' and ''2.4.x' are not covered as old ones.
 @pytest.mark.requires_mgmt
 @pytest.mark.parametrize("manager_version", (
-    "2.6.3",
```

Comment:
This is not exactly skipping, you are removing it from code. which might be that we'll never remember to put it back.
Can you use the skip marker ? with the issue link in it's description ? 
```python
pytest.param('2.6.4', markers=pytest.mark.skip('[link to the issue]')
```

Reply:
Your example is not workable.
Made it be skipped addressing the idea in general.